### PR TITLE
staging release notes and components.yml changes

### PR DESCRIPTION
**Pull request by parthshahva**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,88 @@
...
+
+### Secondary Analysis
+Version(s):
+
+Lira v0.14.0 (was v0.13.2)
+- Update default 10x WDL config parameters in deployment script to reference CellRanger adapter WDL files
+- Update SS2 WDL parameters in deployment script to includes the zarr utils analysis wdl
+- Improve SS2 subscription query to use ontology IDs instead of the text field
+- The scripts for making subscriptions are using the new HCA DCP Auth method to communicate with Data-Storage service now
+- Pipeline-tools v0.34.0 (was v0.28.2)
```
Comment (dshiga) [inline RELEASE_NOTES.md]: Remove `-` here, as this is a new heading, not a bullet point.

File: components.yml [hunk 2]
```diff
@@ -24,9 +24,10 @@ deployed_component_versions:
...
     greenbox-cromwell: 35-0a01d91-SNAP
     greenbox-falcon: v0.1.3
-    greenbox-lira: v0.13.2
+    greenbox-lira: v0.14.0
     greenbox-pipeline-tools: v0.28.2
     greenbox-skylab:
-      smartseq2: smartseq2_v1.0.0
-  data-store: staging-2018-09-19-15-37-11.release
+      smartseq2: smartseq2_v2.0.0
+  data-store: 56da699cc785c7258d67754ec1987c945960f8c0
```
Comment (Bento007) [inline components.yml]: change to `2018-10-11-15-48-22-staging.release`, i'll update the release doc

**Comment by hannes-ucsc** [comment]
See https://github.com/HumanCellAtlas/dcp/pull/151 (it targets this branch).

File: components.yml (hunk 2 above)
Comment (parthshahva) [reply components.yml]: fixed

File: RELEASE_NOTES.md (hunk 1 above)
Comment (parthshahva) [reply RELEASE_NOTES.md]: fixed

**Pull request merged**